lib/evaluate/intrinsics-library.cc

```diff
@@ -66,9 +66,10 @@ static void AddLibmRealHostProcedures(
       {"exp", F{std::exp}, true}, {"gamma", F{std::tgamma}, true},
       {"hypot", F2{std::hypot}, true}, {"log", F{std::log}, true},
       {"log10", F{std::log10}, true}, {"log_gamma", F{std::lgamma}, true},
-      {"mod", F2{std::fmod}, true}, {"sin", F{std::sin}, true},
-      {"sinh", F{std::sinh}, true}, {"sqrt", F{std::sqrt}, true},
-      {"tan", F{std::tan}, true}, {"tanh", F{std::tanh}, true}};
+      {"mod", F2{std::fmod}, true}, {"pow", F2{std::pow}, true},
+      {"sin", F{std::sin}, true}, {"sinh", F{std::sinh}, true},
+      {"sqrt", F{std::sqrt}, true}, {"tan", F{std::tan}, true},
+      {"tanh", F{std::tanh}, true}};
```

Comment:
Adding a comma after the last element of the array initializer (i.e. `}}` -> `},}`) gets clang-format to format these one per line, which makes diffs easier to read.

Reply:
Thanks, the output is much clearer.